crowbar_framework/lib/crowbar/backup/base.rb

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright 2011-2013, Dell
+# Copyright 2013-2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "logger"
+
+module Crowbar
+  class Backup
+    class Base
+      attr_accessor :logger
+
+      def logger
+        @logger ||= ::Logger.new(STDOUT).tap do |logger|
+          logger.level = ::Logger::DEBUG
+        end
+      end
+
+      class << self
+        def restore_files
+          [
+            [
+              "crowbar/client.pem",
+              "/opt/dell/crowbar_framework/config/client.pem"
+            ],
+            [
+              "etc/crowbar.install.key",
+              "/etc/crowbar.install.key"
+            ],
+            [
+              "etc/crowbar.json",
+              "/etc/crowbar/crowbar.json"
+            ],
+            [
+              "etc/hosts",
+              "/etc/hosts"
+            ],
+            [
+              "etc/HOSTNAME",
+              "/etc/HOSTNAME"
+            ],
+            [
+              "etc/network.json",
+              "/etc/crowbar/network.json"
+            ],
+            [
+              "etc/resolv.conf.forwarders",
+              "/etc/resolv.conf"
+            ],
+            [
+              "etc/client.pem",
+              "/etc/chef/client.pem"
+            ],
+            [
+              "etc/webui.pem",
+              "/etc/chef/webui.pem"
+            ],
+            [
+              "etc/validation.pem",
+              "/etc/chef/validation.pem"
+            ],
+            [
+              "root/.gnupg",
+              "/root/.gnupg"
+            ],
+            [
+              "root/.ssh",
+              "/root/.ssh"
+            ],
+            [
+              "tftp/validation.pem",
+              "/srv/tftpboot/validation.pem"
+            ]
+          ]
```

Comment:
The reason why I introduced this mapping was that it was very hard to understood get an overview about the all files in the old way. There are folders in folder in folder which had than the actual data. This concept don't follow the file hierarchy it's more about grouping files in responsibility groups.